questions/qa-address-formats.en.md

```diff
@@ -143,7 +143,7 @@ Not all countries use postal codes. For example, Angola and some other nations d
 
 **Provide additional delivery instructions.** Include dedicated fields for delivery instructions or notes that can accommodate informal addressing information. This might include fields like "Delivery Notes".
 
-**Allow non-ASCII characters.** Many addresses include characters from non-Latin scripts (e.g., Chinese, Japanese, Cyrillic). Use a Unicode encoding (such as UTF-8) in your pages, backend databases, and all software code in between.
+**Allow non-ASCII characters.** Many addresses include characters from non-Latin scripts. Use ASCII addresses for international shipping labels and customs forms, but include native script addresses as supplementary information when possible. Provide native script addresses to local delivery services for the final delivery stage.
```

Comment:
I'm not sure it needs to be ASCII.  I think Latin text is ok.  For example, you don't normally need to write Spanish addresses without accents.

Reply:
fixed